pages/teams/[id].js

```diff
@@ -2,51 +2,60 @@ import { useRouter } from "next/router";
 import useSWR from "swr";
 import styled from "styled-components";
 import Image from "next/image";
-import { StyledLink } from "@/styles";
 import NavBar from "@/Components/NavBar";
+import GamesDisplay from "@/Components/GamesDisplay";
 
 const TeamContainer = styled.div`
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    padding-top: 2.5rem;
-    margin: 0 10px 0 10px;
+  display: flex;
+  flex-direction: row;
```

Comment:
`row` is the default value for `flex-direction` and therefore not necessary to declare it.